Test2.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import UserTask from './__mocks__/Test2.js';
+
+const addAndRemove = new UserTask();
+describe('test localStorage add', () => {
+  test('Add first sample', () => {
+    const addAndRemove = new UserTask('qwerty', false, 1);
+    expect(addAndRemove.add()).toBe(8);
+  });
+  test('Add second sample', () => {
+    const addAndRemove = new UserTask('Azerty', false, 2);
+    expect(addAndRemove.add()).toBe(9);
+  });
+  test('Add third sample', () => {
+    const addAndRemove = new UserTask('new test', false, 3);
+    expect(addAndRemove.add()).toBe(10);
+  });
+  const addAndRemove1 = new UserTask('vitron1', false, 4);
+  addAndRemove1.add();
+  const addAndRemove2 = new UserTask('vitron2', false, 5);
+  addAndRemove2.add();
+  const addAndRemove3 = new UserTask('vitron3', false, 6);
+  addAndRemove3.add();
+  const addAndRemove4 = new UserTask('vitron4', false, 7);
+  addAndRemove4.add();
+  const addAndRemove5 = new UserTask('vitron4', false, 8);
+  addAndRemove5.add();
+  const addAndRemove6 = new UserTask('vitron4', false, 9);
+  addAndRemove6.add();
+  const addAndRemove7 = new UserTask('vitron4', false, 10);
+  addAndRemove7.add();
```

Comment:
- [ Optional] It is good to use [setup and tear down](https://jestjs.io/docs/setup-teardown) jest methods to prepare data for all/each test case and clear after all/each test case. Please take a look the jest documentation following [this](https://jestjs.io/docs/setup-teardown) link.